app/views/users/show.html.erb

```diff
@@ -1 +1,45 @@
-<h1>The profile of individual user</h1>
\ No newline at end of file
+<div class="container">
+  <div class="card mb-3" style="max-width: 800px;">
+    <div class="row g-0">
+        <div class="col-md-4">
+            <img src =<%= @user.photo %> class="img-fluid rounded" alt="Profile picture" style="height: 200px; width: 200px;">
+        </div>
```

Comment:
- [ ] As i mentioned in the user render, kindly make the image a standalone as it is on the wireframe given 👍 
### Your view:
![image](https://github.com/zuhebahmed88091/Blog_App_Rails/assets/50990925/acfbe9dd-63dd-4e47-869e-6fe5e0532a27)

### Wireframe:
![image](https://github.com/zuhebahmed88091/Blog_App_Rails/assets/50990925/1b34d114-3ea4-4766-bee9-90e1f0dbb42a)